src/Components/Todo_list/Components/Todo_list.css

```
@@ -0,0 +1,155 @@
+.content_columns{
+    width: 30%;
+    height: 500px;
+    background-color: #cbf1ff;
+    border-radius: 7px;
+    border: 1px solid gray;
+}
+
+.content_columns_header{
```

Nếu viết theo `BEM` thì phải viết là
```suggestion
.content__columns__header {
```
Còn không hãy viết thành
```suggestion
.content .columns .header {
```